community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,121 @@
+# Introducing message handling with Azure Service Bus message influence in v0.5
+
+[Arcus.Messaging]() has been arround quite a while now. It was created to remove the huge hurdle of constantly having to write the same boilerplate for processing message queues. 
+See [our annoucing post](https://www.codit.eu/blog/announcing-arcus-messaging-background-jobs/) that introduced Arcus.Messaging for the first time for more information.
+
+Starting from v0.4 and updated in v0.5, the [Arcus.Messaging](https://github.com/arcus-azure/arcus.messaging) has got an upgrade in handling messages from Azure Service Bus.
+Previous versions already had set up a message handling system where messages could be routed and processed according to contextual information, but the system wasn't yet mature enough that it provided a way to report back to Azure when the message processing was completed.
+
+Until now.
+
+## Azure Service Bus message operations
+
+Currently, we provide three specific Azure Service Bus functions:
+- **Completing** a message which marks the Service Bus message 'complete' on Azure so no other queue can process it again (this is normally done automatically but can be turned off so you're in full control).
+- **Dead lettering** a message which marks the Service Bus message 'dead lettered' on Azure so it's moved to the dead letter queue.
+- **Abandoning** a message which marks the Service Bus message 'abandonned' on Azure.
+
+## Specific Azure Service Bus message handlers
+
+Because of the way the message handlers are registered and used inside the application, there's no 'callback' function to access the Azure Service Bus.
+That's why we've created two 'message handler templates' to help with this. These templates implements the message handler signature but also provides you with a whole range of specific Azure Service Bus functions that are related to completing the message processing.
```

Comment:
What about this?
```suggestion

That's why we've created two dedicated message handlers for Azure Service Bus which allow you to interact closely with the message itself so that you can influence the message processing.
```